website/docs/react-native/learn/getting-started/create-project.md

```diff
@@ -72,5 +72,5 @@ npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
 
 1. `cd <YourAppName>`（`<YourAppName>`は実際に作成したときの値に変更してください）
 2. `npm install --legacy-peer-deps`
-3. （macOSの場合のみ）`npx pod-install`
+3. （macOSの場合のみ）`npm run pod-install`
```

Comment:
rn-spoiler#v2021.12.0 does not contain `pod-install` script. We must remove tag from template uri or revert this change...

Reply:
Change rn-spoiler-tag to master.
ffdf8817912ffc86346c1db412ef93f56d9ea296